src/main/java/de/blau/android/dialogs/Layers.java

```diff
@@ -719,6 +721,22 @@ private AlertDialog buildImagerySelectDialog(@Nullable final TableRow row, @Null
         return dialog;
     }
 
+    /**
+     * Displays information Dialog for Background/Overlay imagery layers
+     */
+    @Override
+    public void onItemClick(String id) {
+        TileLayerSource layer = TileLayerSource.get(getContext(), id, true);
+        if (layer != null) {
+            LayerInfo f = new ImageryLayerInfo();
+            f.setShowsDialog(true);
+            Bundle args = new Bundle();
+            args.putSerializable(ImageryLayerInfo.LAYER_KEY, layer);
+            f.setArguments(args);
+            LayerInfo.showDialog(getActivity(), f);
+        }
+    }
+
```

Comment:
See above. as we might want to rip the whole imagery selection dialog out and put it in a separate class one day, I would tend to keep everything together.